src/Utils/General/FileSystemHelper.php

```diff
@@ -83,4 +83,41 @@ public static function truncateDir($path)
             }
         }
     }
+    
+    /**
+     * Copy a file, or recursively copy a folder and its contents
+     * @see https://stackoverflow.com/questions/2050859/copy-entire-contents-of-a-directory-to-another-using-php/2050909#2050909
+     *
+     * @param       string   $source    Source path
+     * @param       string   $dest      Destination path
+     * @return      bool     Returns TRUE on success, FALSE on failure
+     */
+    public static function copyFiles($source, $dest)
```

Comment:
See my comment on https://github.com/Learnosity/learnosity-qti/pull/35/files#diff-4f6fbaca35fc195db5a5bcb196b7a536R14.

Let's keep this one and remove the other, no need for two versions of the same code.

Reply:
Done